generated/nidaqmx/nidaqmx.proto

```diff
@@ -3066,8 +4949,8 @@ enum GpsSignalType1 {
 }
 
 enum GroupBy {
-  GROUP_BY_GROUP_BY_CHANNEL = 0;
-  GROUP_BY_GROUP_BY_SCAN_NUMBER = 1;
+  GROUP_BY_CHANNEL = 0;
+  GROUP_BY_SCAN_NUMBER = 1;
 }
```

Comment:
This breaks source compatibility. (However, if it didn't, I would prefer the shorter names.)

Reply:
Sure Brad. So, are we good with this change? or we should fix this as well?

Reply:
No. This change is not okay. It breaks the tests and most of the examples, so it will also break most clients of nidaqmx.proto. Did you run any grpc-device DAQmx tests or examples?

```
PS C:\dev\grpc-device> rg GROUP_BY_GROUP_BY
source\tests\system\nidaqmx_driver_api_tests.cpp
261:    request.set_fill_mode(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
276:    request.set_data_layout(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
287:    request.set_data_layout(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
303:    request.set_fill_mode(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
312:    request.set_data_layout(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
328:    request.set_fill_mode(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
337:    request.set_data_layout(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
351:    request.set_fill_mode(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
362:    request.set_data_layout(GroupBy::GROUP_BY_GROUP_BY_CHANNEL);
1148:      GroupBy::GROUP_BY_GROUP_BY_CHANNEL,
1178:      GroupBy::GROUP_BY_GROUP_BY_CHANNEL,

examples\nidaqmx\analog-output.py
83:            data_layout=nidaqmx_types.GROUP_BY_GROUP_BY_CHANNEL,

examples\nidaqmx\digital-output.py
81:            data_layout=nidaqmx_types.GROUP_BY_GROUP_BY_CHANNEL,

examples\nidaqmx\dsa-shared-timebase-and-trig-analog-input-and-output.py
386:                    data_layout=nidaqmx_types.GROUP_BY_GROUP_BY_CHANNEL,
398:                    data_layout=nidaqmx_types.GROUP_BY_GROUP_BY_CHANNEL,
469:                                fill_mode=nidaqmx_types.GroupBy.GROUP_BY_GROUP_BY_CHANNEL,
482:                                fill_mode=nidaqmx_types.GroupBy.GROUP_BY_GROUP_BY_CHANNEL,

examples\nidaqmx\digital-input.py
80:            fill_mode=nidaqmx_types.GROUP_BY_GROUP_BY_CHANNEL,

examples\nidaqmx\analog-input.py
105:            fill_mode=nidaqmx_types.GROUP_BY_GROUP_BY_CHANNEL,

examples\nidaqmx\analog-input-every-n-samples.py
135:                                    fill_mode=nidaqmx_types.GroupBy.GROUP_BY_GROUP_BY_CHANNEL,
```

Reply:
Added a fix in https://github.com/ni/grpc-device-scrapigen/pull/127/commits/2ff22d5b025bf6287e43aff545216a1ba34eca4a to remove function that was doing this explicitly